dip-template/pallets/nft-marketplace/src/lib.rs

```diff
@@ -250,30 +241,25 @@ pub mod pallet {
 		/// The maximum length of data stored in for post codes.
 		#[pallet::constant]
 		type PostcodeLimit: Get<u32>;
+
+		type DidIdentifier: Parameter + MaxEncodedLen;
```

Comment:
This type could also be hardcoded somewhere else instead of being a type parameter of this crate.